### PR TITLE
Fixup net_tutorial.md

### PR DESCRIPTION
**Pull request by andrum99**
This is a new PR to update instructions for net boot based on #1930, since the author of that PR (@Malvineous) has not yet responded to requests to finish that PR and #1911 is looming.

I've also fixed some minor grammar issues - some language which was rather informal for a technical document.

### CODE REVIEW

**Comment by aallan** [comment]
Okay, so either this PR or #1930 should be merged?

**Comment by andrum99** [comment]
> Okay, so either this PR or #1930 should be merged?

Yes. This is the only way I can get it done. Alternatively someone with write access to this repo can integrate #1930 then make the fixups on master directly. Or the author of #1930 fixes it up in time of course.

File: hardware/raspberrypi/bootmodes/net_tutorial.md [hunk 1]
```diff
@@ -1,24 +1,20 @@
 # Network boot your Raspberry Pi
 
-This tutorial is written to explain how to set up a simple DHCP/TFTP server which will allow you to boot a Raspberry Pi 3 from the network. The tutorial assumes that you have an existing home network, and that you want to use a Raspberry Pi for the **server**. You will need a second Raspberry Pi 3 as a **client** to be booted. Only one SD card is needed because the client will be booted from the server after the initial client configuration.
+This tutorial is written to explain how to set up a simple DHCP/TFTP server which will allow you to boot a Raspberry Pi 3 or 4 from the network. The tutorial assumes that you have an existing home network, and that you want to use a Raspberry Pi for the **server**. You will need a second Raspberry Pi 3 as a **client** to be booted. Only one SD card is needed because the client will be booted from the server after the initial client configuration.
```
Comment (lurch) [inline hardware/raspberrypi/bootmodes/net_tutorial.md:3]: I guess "a second Raspberry Pi 3" should be "a second Raspberry Pi 3 or Raspberry Pi 4" ? (and perhaps "second" should be "separate"?)

Comment (andrum99) [reply hardware/raspberrypi/bootmodes/net_tutorial.md]: > I guess "a second Raspberry Pi 3" should be "a second Raspberry Pi 3 or Raspberry Pi 4" ? (and perhaps "second" should be "separate"?)

Good spot - fixed.

**Review by lurch — APPROVED** [review]
LGTM

**Pull request merged**